ocs_ci/templates/workloads/vdbench/vdbench_full.yaml

```diff
@@ -0,0 +1,44 @@
+apiVersion: ripsaw.cloudbulldozer.io/v1alpha1
```

Comment:
I guess one file can be provided and remaining parameters can be updated depending on the test.

Reply:
This is correct, you can pass a template file as parameter, and also change parameters from the template.